plugin/src/main/java/com/denizenscript/denizen/objects/PlayerTag.java

```diff
@@ -3875,14 +3924,30 @@ else if (getPlayerEntity().getGameMode() == GameMode.SPECTATOR) {
         // @name sign_update
         // @input ElementTag
         // @description
-        // Shows the player fake lines on a sign, with input in the format of LocationTag|ListTag.
+        // Shows the player fake lines on a sign, with input in the format of LocationTag|ListTag
+        // or LocationTag|MapTag, with keys front and/or back.

```

Comment:
Using `ListTag`s like this is usually a legacy thing that's replaced by `MapTag`s, so the modern format should be a map with `location`/`front`/`back` keys (could split it up to front/back like other mechanisms, but since this is already a map input anyway, probably fine to keep it all in one mech)